docs/iam-usage.rst

```diff
@@ -0,0 +1,144 @@
+Using IAM For Users
+===================
+
+.. note:: This document does not explain how Google Cloud IAM works,
```

Comment:
Maybe change this to something like..."This document assumes knowledge of concepts and patterns described in..."

Just a thought.